pom.xml

```diff
@@ -116,6 +116,7 @@
         <module>webtau-junit5</module>
         <module>webtau-junit5-groovy</module>
         <module>webtau-junit5-examples</module>
+        <module>webtau-junit5-examples-kotlin</module>
```

Comment:
`junit5-examples` has both groovy and java ones. Is there an issue to put Kotlin there as well?

Reply:
I had a hard time getting Maven to build all three but I've found an SO article that suggests it's possible so happy to give it another go.

Reply:
That would help. If not possible, probably should make a parent pom to avoid repetition

Reply:
Yeah, makes sense.

Reply:
I figured it out.